src/bin/vhost_user_fs.rs

```diff
@@ -30,11 +31,13 @@ use virtio_bindings::bindings::virtio_net::*;
 use virtio_bindings::bindings::virtio_ring::{
     VIRTIO_RING_F_EVENT_IDX, VIRTIO_RING_F_INDIRECT_DESC,
 };
-use vm_memory::GuestMemoryMmap;
+use vm_memory::{GuestAddressSpace, GuestMemoryAtomic, GuestMemoryMmap};
+use vm_virtio::queue::DescriptorChain;
 use vmm_sys_util::eventfd::EventFd;
 
 const QUEUE_SIZE: usize = 1024;
 const NUM_QUEUES: usize = 2;
+const THREAD_POOL_SIZE: usize = 64;
```

Comment:
How did you choose this number?

Reply:
That value comes from the usual queue depth of block devices. virtiofsd and QEMU's thread pool use the same number.